README.md

```diff
@@ -64,3 +68,32 @@ For more information, see the [Rust API embedding chapter] of the Wasmtime docum
 [Rust API embedding chapter]: https://bytecodealliance.github.io/wasmtime/embed-rust.html
 
 It's Wasmtime.
```

Comment:
The "It's Wasmtime"  line applies to the project as a whole not just the embedding API, so we should keep it at the bottom of the file :-).